src/main/scala/play/dev/filewatch/FileWatchService.scala

```diff
@@ -92,18 +97,31 @@ object FileWatchService {
       override def toString = delegate.toString
     }
 
+  def defaultWatchService(targetDirectory: File, pollDelayMillis: Int, logger: LoggerProxy): FileWatchService =
+    defaultWatchService(targetDirectory, pollDelayMillis, logger, disableFileHashCheck = false)
+
   def jnotify(targetDirectory: File): FileWatchService = optional(JNotifyFileWatchService(targetDirectory))
 
-  def jdk7(logger: LoggerProxy): FileWatchService = default(logger, isMac = false)
+  def jdk7(logger: LoggerProxy, disableFileHashCheck: Boolean): FileWatchService =
+    default(logger, isMac = false, disableFileHashCheck)
 
-  def mac(logger: LoggerProxy): FileWatchService = default(logger, isMac = true)
+  def jdk7(logger: LoggerProxy): FileWatchService = jdk7(logger, disableFileHashCheck = false)
+
+  def mac(logger: LoggerProxy, disableFileHashCheck: Boolean): FileWatchService =
+    default(logger, isMac = true, disableFileHashCheck)
+
+  def mac(logger: LoggerProxy): FileWatchService = mac(logger, disableFileHashCheck = false)
 
   def polling(pollDelayMillis: Int): FileWatchService = new PollingFileWatchService(pollDelayMillis)
 
   def optional(watchService: Try[FileWatchService]): FileWatchService =
     new OptionalFileWatchServiceDelegate(watchService)
 
-  def default(logger: LoggerProxy, isMac: Boolean): FileWatchService = new DefaultFileWatchService(logger, isMac)
+  def default(logger: LoggerProxy, isMac: Boolean, disableFileHashCheck: Boolean): FileWatchService =
+    new DefaultFileWatchService(logger, isMac, disableFileHashCheck)
+
+  def default(logger: LoggerProxy, isMac: Boolean): FileWatchService =
+    default(logger, isMac, disableFileHashCheck = true)
```

Comment:
`true` by default? Shouldn't this be `disableFileHashCheck = false`?

Reply:
as I just commented the idea was to not change anything, but allow a change in play or by users...

Reply:
ah ofc
```suggestion
    default(logger, isMac, disableFileHashCheck = false)
```